Dockerfiles/Dockerfile.package

```diff
@@ -7,6 +7,8 @@ ARG GPG_RPM_EMAIL
 ARG BCDA_GPG_RPM_PASSPHRASE
 
 RUN apt-get update
+RUN apt-get install -y build-essential ruby ruby-dev rpm git
```

Comment:
Can we safely remove `ruby` and `ruby-dev` here, and the following line?

Reply:
> Can we safely remove `ruby` and `ruby-dev` here, and the entire next line?

No, these are necessary for installing `fpm`, which we use to build the RPM.  The main reason for this image is so we can spin up a container to build an RPM (and not have to rely on the dependencies existing on Jenkins and/or local).  
Reference of using `fpm` in `bcda-ssas-app`: https://github.com/CMSgov/bcda-ssas-app/blob/master/ops/build_and_package.sh#L31
Reference of the similar image in `bcda-app`: https://github.com/CMSgov/bcda-app/blob/master/Dockerfiles/Dockerfile.package#L10-L11